src/python/sdk/examples/trade.py

```diff
@@ -35,62 +35,71 @@
 """
 HOW TO RUN THIS SCRIPT: `poetry install && poetry run trade` in .../econia/src/python/sdk
 
-RECOMMENDED: Use a local development chain (else you might hit rate-limiting issues)
-1. Run: aptos node run-local-testnet --with-faucet
-2. Do: "Deploy an Econia Faucet" (above, enter "local" for the `aptos init` chain)
-3. Do: "Deploy an Econia Exchange"
-2. Enter: http://0.0.0.0:8080/v1 when prompted node URL
-3. Enter: http://0.0.0.0:8081 when prompted for a faucet URL
-
-REQUIRED: Deploy an Econia Faucet.
-1. Run: cd .../econia/src/move/faucet # (whatever its full path is)
-2. Run: aptos init --profile econia_faucet_deploy
-3. Run: export FAUCET_ADDR=<ACCOUNT-FROM-ABOVE> # make sure to put 0x at the start
-4. Run: aptos move publish \
-        --named-addresses econia_faucet=$FAUCET_ADDR \
-        --profile econia_faucet_deploy \
-        --assume-yes
-
-OPTIONAL: Deploy an Econia Exchange.
-1. Run: cd /econia/src/move/econia # (or whatever its full path is)
-2. Run: aptos init --profile econia_exchange_deploy
-3. Run: export ECONIA_ADDR=<ACCOUNT-FROM-ABOVE> # make sure to put 0x at the start
-4. Run: aptos move publish \
-        --override-size-check \
-        --included-artifacts none \
-        --named-addresses econia=$ECONIA_ADDR \
-        --profile econia_exchange_deploy \
-        --assume-yes
+See instructions in /src/docker/README.md to see how to run this script against a local
+deployment of Econia (including the backend service stack).
+
+There are several prompts; entering nothing for all of them will default to running
+against a local deployment without performing a market recognization.
```

Comment:
The final part of the sentence is confusing, and it's not immediately clear if "local deployment" means a local testnet from a CLI or from the local version of the DSS

How about?

```
There are several prompts; entering nothing for all of them will result in:
- Running against a local deployment of the data service stack.
- No market will be listed as recognized.
```